### DISCUSSION TITLE
My university won't issue me an email

### DISCUSSION THREAD

**Discussion by ezeamin** [community/community#54391] [GitHub Education]
### Hi everyone,

Hi! I'm studying at a private university from Tucumán, Argentina, but they won't give me an academic email (because it's just for people who work there). I have all affiliation proof needed for applying, but the system requires I use an academic email.

My university is called UNSTA (Universidad del Norte Santo Tomás de Aquino).

I've talked with the university administrators, and they told me it was impossible to give me one, since it's only for them.

None student has one.

I currently have my GitHub education account active, but not for much longer. This new requirement is cutting me off!

Thanks in advance!

**Comment by samus-aran** [comment]
Hi @ezeamin  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
